crates/ruff_python_formatter/src/other/identifier.rs

```diff
@@ -54,14 +56,16 @@ impl<'a> DotDelimitedIdentifier<'a> {
 
 impl Format<PyFormatContext<'_>> for DotDelimitedIdentifier<'_> {
     fn fmt(&self, f: &mut PyFormatter) -> FormatResult<()> {
-        // An import identifier can contain newlines by inserting continuations (backslashes) after
-        // a dot-delimited segment, as in:
-        // ```python
-        // import foo\
-        //    .bar
-        // ```
-        if memchr::memchr(b'\\', f.context().source()[self.0.range()].as_bytes()).is_some() {
-            text(self.0.as_str(), Some(self.0.start())).fmt(f)
+        // The backslashes are line continuations
```

Comment:
Can we restore the existing comment, and add a second example to explain that there can be arbitrary whitespace between dots? The new comment feels like a less-useful subset of the existing comment.